test/test_commit.py

```diff
@@ -149,6 +149,81 @@ def check_entries(d):
++Please note that the script must be executable (i.e. `chmod +x script.sh`). `StrictHostKeyChecking=no` is used to avoid prompts asking to save the hosts key to `~/.ssh/known_hosts`, which happens in case you run this as daemon.
+ 
+ You might also have a look at `Git.update_environment(...)` in case you want to setup a changed environment more permanently.
+ 
+@@ -509,14 +509,14 @@ The type of the database determines certain performance characteristics, such as
+ 
+ GitDB
+ =====
+-The GitDB is a pure-python implementation of the git object database. It is the default database to use in GitPython 0.3. Its uses less memory when handling huge files, but will be 2 to 5 times slower when extracting large quantities small of objects from densely packed repositories::
++The GitDB is a pure-python implementation of the git object database. It is the default database to use in GitPython 0.3. It uses less memory when handling huge files, but will be 2 to 5 times slower when extracting large quantities of small objects from densely packed repositories::
+ 
+     repo = Repo("path/to/repo", odbt=GitDB)
+ 
+ 
+ GitCmdObjectDB
+ ==============
+-The git command database uses persistent git-cat-file instances to read repository information. These operate very fast under all conditions, but will consume additional memory for the process itself. When extracting large files, memory usage will be much higher than the one of the ``GitDB``::
++The git command database uses persistent git-cat-file instances to read repository information. These operate very fast under all conditions, but will consume additional memory for the process itself. When extracting large files, memory usage will be much higher than ``GitDB``::
+ 
+     repo = Repo("path/to/repo", odbt=GitCmdObjectDB)
+ """
+        commit = self.rorepo.commit('c0740570b31f0f0fe499bf4fc5abbf89feb1757d')
+        patch = commit.patch
+        self.assertEqual(patch[:200], expected_path_result[:200])
+        self.assertEqual(patch[200:400], expected_path_result[200:400])
+        self.assertEqual(patch[400:600], expected_path_result[400:600])
+
+    def test_patch_with_no_parent(self):
+        expected_path_result = """diff --git a/.flake8 b/.flake8
+new file mode 100644
+index 00000000..c55fe35d
+--- /dev/null
++++ b/.flake8
+@@ -0,0 +1,35 @@
++[flake8]
++show-source = True
++count= True
++statistics = True
++# E265 = comment blocks like @{ section, which it can't handle
++# E266 = too many leading '#' for block comment
++# E731 = do not assign a lambda expression, use a def
++# W293 = Blank line contains whitespace
++# W504"""
+        commit = self.rorepo.commit('33ebe7acec14b25c5f84f35a664803fcab2f7781')
+        patch = commit.patch
+        self.assertEqual(patch[:200], expected_path_result[:200])
```

Comment:
As above, I don't think the slice syntax should be necessary here. If it is, a description on why that is would be necessary, it could be passed as third argument to `assertEqual` I believe.

Reply:
have to set maxDiff = None to disable limit and test. I will do the change and test it.